clients/gtest/geqrf_strided_batched_gtest.cpp

```diff
@@ -82,7 +82,51 @@ TEST_P(geqrf_strided_batched_gtest, geqrf_strided_batched_gtest_double)
 
     Arguments arg = setup_geqrf_strided_batched_arguments(GetParam());
 
-    hipblasStatus_t status = testing_geqrf_strided_batched<double>(arg);
+    hipblasStatus_t status = testing_geqrf_strided_batched<double, double>(arg);
+
+    if(status != HIPBLAS_STATUS_SUCCESS)
+    {
+        if(arg.M < 0 || arg.N < 0 || arg.lda < arg.M || arg.batch_count < 0)
+        {
+            EXPECT_EQ(HIPBLAS_STATUS_INVALID_VALUE, status);
+        }
+        else
+        {
+            EXPECT_EQ(HIPBLAS_STATUS_NOT_SUPPORTED, status); // for cuda
```

Comment:
CUDA doesn't support strided_batched correct? If so this is good :)

Reply:
As far as I know, yes.